cypress/e2e/timeline/timeline.spec.ts

```diff
@@ -250,7 +250,7 @@ describe("Timeline", () => {
             cy.contains(".mx_RoomView_body .mx_EventTile[data-scroll-tokens]", "MessageEdit").should("exist");
 
             // Click top left of the event toggle, which should not be covered by MessageActionBar's safe area
-            cy.get(".mx_EventTile .mx_ViewSourceEvent")
+            cy.get(".mx_EventTile:not(:first-child) .mx_ViewSourceEvent")
```

Comment:
A comment here to explain this might be helpful - I assume it's because of "show hidden events" as we discussed?